client/src/lib/error-telemetry.js

```diff
@@ -61,9 +76,9 @@ class ErrorTelemetry {
       logger.debug('Reading line:', data);
```

Comment:
If some of the reasoning behind `outputFunction` is to scope into this callback, then I suggest looking at the re-assignment of `this` to `self` which is used in other places for just this purpose